### ISSUE TITLE
Anyway to change the mod hotkey?

### ISSUE THREAD

**Issue by Olliesama** [OrfeasZ/ZHMModSDK#144]
Heya, great mod so far and am using the version found on freelancer variations until the official release is out, before I used noui=true so it wasn't an issue but I use the key for the mod menu for the inventory, is there a method to change this hotkey?

**Comment by Asrda** [comment]
Sorry i dont have the answer to your question but how is SDK even working for you right now? The one found on freelancer variations is just the one found here?

**Comment by Olliesama** [comment]
> Sorry i dont have the answer to your question but how is SDK even working for you right now? The one found on freelancer variations is just the one found here?

https://www.nexusmods.com/hitman3/mods/437?tab=posts
I just used the one posted there.

**Comment by kellne** [comment]
> > Sorry i dont have the answer to your question but how is SDK even working for you right now? The one found on freelancer variations is just the one found here?
> 
> https://www.nexusmods.com/hitman3/mods/437?tab=posts I just used the one posted there.


it works man. But is there working version with Free Camera stopping time ? I like it for photos. This link works but time doest stop :(

**Comment by OrfeasZ** [comment]
@Olliesama yes. See: https://github.com/OrfeasZ/ZHMModSDK/issues/64#issuecomment-2081680921
